provider/sql.go

```diff
@@ -1269,8 +1269,9 @@ func (q defaultOfflineSQLQueries) createValuePlaceholderString(columns []TableCo
 func (q defaultOfflineSQLQueries) trainingSetQuery(store *sqlOfflineStore, def TrainingSetDef, tableName string, labelName string, isUpdate bool) error {
 	columns := make([]string, 0)
 	query := ""
+	lagFeaturesOffset := 0
```

Comment:
can we just move this down and do  `lagFeaturesOffset := len(def.Features)`

Reply:
smart thinking, changed